9/Dockerfile

```diff
@@ -61,6 +62,7 @@ RUN mkdir -p /var/run/redis && \
     chmod +x /openvas-check-setup && \
     sed -i 's/DAEMON_ARGS=""/DAEMON_ARGS="-a 0.0.0.0"/' /etc/init.d/openvas-manager && \
     sed -i 's/DAEMON_ARGS=""/DAEMON_ARGS="--mlisten 127.0.0.1 -m 9390 --gnutls-priorities=SECURE128:-AES-128-CBC:-CAMELLIA-128-CBC:-VERS-SSL3.0:-VERS-TLS1.0"/' /etc/init.d/openvas-gsa && \
+    sed -i '/^DAEMON_ARGS="/aDAEMON_ARGS="$DAEMON_ARGS --allow-header-host=$PUBLIC_HOSTNAME"' /etc/init.d/openvas-gsa && \
```

Comment:
It's intentional, to insert text below the matched line:
```
$ cat example
hello
goodbye
$ sed '/hello/ahowareyou' example
hello
howareyou
goodbye
```
Verified on Ubuntu 16.04 and also in the container when I was building the patch.

I did this because the single DAEMON_ARGS assignment was getting absurdly long.

Also, in theory we'd want to handle multiple --allow-header-host= (it's allowed to repeat them) in which case repeating the sed command for each entry in the env var would work (with some added looping logic)

Reply:
Ahh yeah that makes sense, hadn't had my coffee yet. It was too early for regex!

Thanks!